# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1251266545 -7200
#      Wed Aug 26 08:02:25 2009 +0200
# Node ID b8db3595f706ad559ec4c5e803764d4e4228cf4a
# Parent  c60a9e1a03727db13a5a8670dd553b7b5f7d487c
fix order of args in fmod

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2009-08-26  Jaroslav Hajek  <highegg@gmail.com>
+
+	* data.cc (Ffmod): Reverse order of args.
+
 2009-08-25  John W. Eaton  <jwe@octave.org>
 
 	* graphics.cc (gnuplot_backend::send_quit): Wait for gnuplot process.
 
 2009-08-24  Jaroslav Hajek  <highegg@gmail.com>
 
 	* OPERATORS/op-m-m.cc: Install .*= and ./= operators.
 	* OPERATORS/op-fm-fm.cc: Ditto.
diff --git a/src/data.cc b/src/data.cc
--- a/src/data.cc
+++ b/src/data.cc
@@ -1101,18 +1101,18 @@ sign as @var{x}.  If @var{y} is zero, th
 @end deftypefn")
 {
   octave_value retval;
 
   int nargin = args.length ();
 
   if (nargin == 2 && args(0).is_defined () && args(1).is_defined ())
     {
-      octave_value arg_y = args(0);
-      octave_value arg_x = args(1);
+      octave_value arg_x = args(0);
+      octave_value arg_y = args(1);
 
       dim_vector y_dims = arg_y.dims ();
       dim_vector x_dims = arg_x.dims ();
 
       bool y_is_scalar = y_dims.all_ones ();
       bool x_is_scalar = x_dims.all_ones ();
 
       bool is_float = arg_y.is_single_type () || arg_x.is_single_type ();
